channels/us.m3u

```diff
@@ -3445,75 +3445,75 @@ https://dai.google.com/linear/hls/event/Xu-I6qGiSTKeu6glzS4BtQ/master.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-tvsmystery/tracks-v1a1/index-1623399442-now.m3u8
 #EXTINF:-1 tvg-id="TVSClassicMovies.us" tvg-name="TVS Classic Movies" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/VLWDHGs.png" group-title="Classic",TVS Classic Movies (480p)
-https://nrpus.bozztv.com/36bay2/gusa-tvsclassicmovies/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-tvsclassicmovies/tracks-v1a1/index-1623399888-now.m3u8
 #EXTINF:-1 tvg-id="TVSClassicSports.us" tvg-name="TVS Classic Sports" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ui099xQ.jpg" group-title="Sports",TVS Classic Sports (720p)
-https://nrpus.bozztv.com/36bay2/gusa-tvs/index.m3u8
+http://rpn1.bozztv.com/36bay2/gusa-tvs/tracks-v1a1/index-1623399962-now.m3u8
 #EXTINF:-1 tvg-id="TVSComedyNetwork.us" tvg-name="TVS Comedy Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/lHvWCyV.jpg" group-title="Comedy",TVS Comedy Network (480p)
-https://nrpus.bozztv.com/36bay2/gusa-tvscomedy/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-comedyclassics/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="TVSDriveInMovie.us" tvg-name="TVS Drive In Movie" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RhKSIGQ.jpg" group-title="Movies",TVS Drive In Movie (360p)
-https://nrpus.bozztv.com/36bay2/gusa-tvsdriveinmovie/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-tvsdriveinmovie/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="TVSFamilyChannel.us" tvg-name="TVS Family Channel" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/iuQyJ7f.jpg" group-title="Family",TVS Family Channel (360p)
-https://nrpus.bozztv.com/36bay2/gusa-TVSFamilyChannel/index.m3u8
-#EXTINF:-1 tvg-id="TVSFilmNoirNetwork.us" tvg-name="TVS Film Noir Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Film Noir Network (360p)
-https://nrpus.bozztv.com/36bay2/gusa-TVSFilmNoir/index.m3u8
-#EXTINF:-1 tvg-id="TVSFrontPageDetective.us" tvg-name="TVS Front Page Detective" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Front Page Detective (360p)
-https://nrpus.bozztv.com/36bay2/gusa-tvsfrontpagedetective/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-TVSFamilyChannel/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="TVSFilmNoirNetwork.us" tvg-name="TVS Film Noir Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Movies",TVS Film Noir Network (360p)
+https://rpn1.bozztv.com/36bay2/gusa-TVSFilmNoir/tracks-v1a1/index-1623400693-now.m3u8
+#EXTINF:-1 tvg-id="TVSFrontPageDetective.us" tvg-name="TVS Front Page Detective" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Front Page Detective (360p)
+https://rpn1.bozztv.com/36bay2/gusa-tvsfrontpagedetective/tracks-v1a1/index-1623400775-now.m3u8
```

Comment:
Better use this :  https://rpn1.bozztv.com/36bay2/gusa-tvsfrontpagedetective/tracks-v1a1/mono.m3u8